subsys/net/lib/lwm2m_client_utils/Kconfig

```diff
@@ -132,6 +113,25 @@ config LWM2M_CLIENT_UTILS_LOCATION_ASSIST_AGPS_BUF_SIZE
 	depends on LWM2M_CLIENT_UTILS_LOCATION_ASSIST_AGPS
 	default 4096
 
+config LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT
+	bool "Signal measurement information object support (ID 10256)"
+	depends on LWM2M_CLIENT_UTILS_LOCATION_ASSIST_CELL
```

Comment:
In ATv2 we are not using any functionality that is guarded by `CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_CELL`, we only need the `ecid_signal_meas_info.c` source file and its APIs through the `lwm2m_client_utils.h` header file. So maybe just remove the dependency here since it can be used independently. Then we don't need `CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_CELL=y` set in `overlay-lwm2m.conf`.

Reply:
Hmm, need to think it over. Thanks.